src/modules/Elsa.Retention/Extensions/WorkflowInstanceFilterExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using Elsa.Workflows;
+using Elsa.Workflows.Management.Filters;
+using Elsa.Workflows.Management.Models;
+
+namespace Elsa.Retention.Extensions;
+
+public static class WorkflowInstanceFilterExtensions
+{
+
+    /// <summary>
+    /// Clone the current filter
+    /// </summary>
+    /// <param name="filter"></param>
+    /// <returns></returns>
```

Comment:
Let's remove empty XML comments (with the exception of params where a comment was provided for at least one, in which case param comments should be present for the others).